### DISCUSSION TITLE
How to show images in a module type extension

### DISCUSSION THREAD

**Discussion by Juanjo-GEx** [directus/directus#15534] [Feature Requests]
What is the correct way to display images in a Directus module?
Using the following code:

```vue
<template>
	<private-view title="Components">
		<v-list>
			<v-list-item v-for="item in items" :key="item.id">
				<v-img :src="'http://localhost:8055/assets/' + item.id" :alt="item.title" />
			</v-list-item>
		</v-list>
	</private-view>
</template>
```
The src is generated fine but it doesn't load the image and it is not displayed in the module:

![image](https://user-images.githubusercontent.com/110392472/189651321-e16d8ded-59a9-42cf-bce9-6ae412ce6edb.png)

**Comment by br41nslug** [comment]
I think you meant to use `<v-image` there that should resolve it 😄

**Reply by Juanjo-GEx** [reply]
I have lost an hour of my life for this clumsiness :( 
Thanks a lot!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
